mono/metadata/debug-mono-ppdb.c

```diff
@@ -125,8 +126,10 @@ mono_ppdb_load_file (MonoImage *image, const guint8 *raw_contents, int size)
 		return create_ppdb_file (image);
 	}
 
-	if (!get_pe_debug_guid (image, pe_guid, &pe_age, &pe_timestamp))
+	if (!get_pe_debug_guid (image, pe_guid, &pe_age, &pe_timestamp)) {
+		mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_ASSEMBLY, "Image '%s' has no debug directory.", image->name);
```

Comment:
How about other cases for failure of loading debug info, specially if --debug was passed?